src/static/css/content.less

```diff
@@ -316,6 +322,19 @@ figure {
 // Initiate loop
 .color-swatch-loop( length(@colors) );
 
+.color-table {
+    width: 100%;
+    margin-bottom: 40px;
+
+    .swatch_field {
+        width: 200px;
+        height: 2.5em;
```

Comment:
I think we should keep the gray only because you can't tell the white is even there without the border above/below.